lib/engines/content_block_manager/app/services/content_block_manager/publish_edition_service.rb

```diff
@@ -28,6 +30,7 @@ def publish_with_rollback(content_block_edition)
         },
       )
       ContentBlockManager::SchedulePublishingWorker.dequeue(content_block_edition) if content_block_edition.scheduled?
```

Comment:
> ContentBlockManager::SchedulePublishingWorker.dequeue(content_block_edition) if content_block_edition.scheduled?

do you think this line could be part of the dequeuing concern? or is the dequeuing concern already doing this?

Reply:
or could it have it's own named function to show what it's doing like `dequeue_current_edition_if_previously_scheduled`

Reply:
Good plan! Have done 👍